### DISCUSSION TITLE
ServiceConnection how to

### DISCUSSION THREAD

**Discussion by alexanderankin** [awspring/spring-cloud-aws#1178] [Q&A]
Hi all,

I tried to use ServiceConnection and found this https://repo.maven.apache.org/maven2/io/awspring/cloud/spring-cloud-aws-testcontainers/3.2.0-M1/

I was wondering if this is the right one to use as it doesn't look like a final release.

thanks.

**Comment by maciejwalkowiak** [comment]
We have just released 3.2.0 with `@ServiceConnection` support. Give it a go, feedback appreciated!

**Reply by alexanderankin** [reply]
amazing news, I most certainly will
